code/game/objects/effects/meteors.dm

```diff
@@ -194,6 +194,50 @@ GLOBAL_LIST_INIT(meteors_gore, list(/obj/effect/meteor/meaty = 5, /obj/effect/me
 //Meteor types
 ///////////////////////
 
+//Fake
+/obj/effect/meteor/fake
+	name = "simulated meteor"
+	desc = "A simulated meteor for testing shield satellites. How did you see this, anyway?"
+	invisibility = INVISIBILITY_MAXIMUM
+	density = FALSE
+	pass_flags = NONE
+	var/datum/station_goal/station_shield/goal
+	var/failed = FALSE
```

Comment:
```suggestion
	/// The station goal that is simulating this meteor.
	var/datum/station_goal/station_shield/goal
	/// Did we crash into something? Used to avoid falsely reporting success when qdeleted.
	var/failed = FALSE
```